### ISSUE TITLE
Lost WYSIWG changes inside a translation component when user changes the language

### ISSUE THREAD

**Issue by joancarlesbadia** [hanneskuettner/directus-extension-group-tabs-interface#11]
When you use a tab-group with tabs with WYSWG components and are inside a one translation component, lost content when change language.

Thanks,

**Comment by hanneskuettner** [comment]
Can you provide a quick screen recording and example setup for this problem?

**Comment by joancarlesbadia** [comment]
[Enregistrament de pantalla 2024-07-03 092152.zip](https://github.com/user-attachments/files/16082256/Enregistrament.de.pantalla.2024-07-03.092152.zip)
You can see on top one WYSIWYG that we have got out from the tabs to compare with the bottom tabs.
![image](https://github.com/hanneskuettner/directus-extension-group-tabs-interface/assets/108670131/ceea3872-c9f5-4dfc-b1c5-19fc4c6d2dee)

all the WYSIWYG are equally defined:
![image](https://github.com/hanneskuettner/directus-extension-group-tabs-interface/assets/108670131/ad0b6706-6799-4cb9-b900-f203001d3b44)

**Comment by hanneskuettner** [comment]
Oh so you have tabs nested in tabs? I'll have to look into that.

Also... you got me good with the Slack notification sounds in the video... lol

**Comment by joancarlesbadia** [comment]
Yes, maybe nested tabs inside it's unnecessary. We have tested also with only one level and is also failing .
Thank you for your quick response.

**Comment by bryantgillespie** [comment]
I noticed this and I think it may be the same / similar issue.

Directus holds on to the value for relational fields, but when switching tabs the value is lost in the components within.

https://github.com/user-attachments/assets/b6257bb8-b675-4a1d-aa68-ceb6ba5e7ac7

**Setup**
- tab group
  - raw group
    - field
  - m2o field

![ScreenShot 2024-07-20 at 11 05 09@2x](https://github.com/user-attachments/assets/67ab0de6-543a-477d-959d-3a28f686a4f6)
